hrm-domain/hrm-service/src/data-pull-task/khp-data-pull-task/pull-contacts.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+import format from 'date-fns/format';
+import formatISO from 'date-fns/formatISO';
+import parseISO from 'date-fns/parseISO';
+
+import { getContext, noLimitsOrOffset, maxPermissions } from './context';
+import * as contactApi from '../../contact/contact';
+
+const getSearchParams = (startDate: Date, endDate: Date) => ({
+  dateFrom: formatISO(startDate),
+  dateTo: formatISO(endDate),
+  onlyDataContacts: false,
+});
+
+export const pullContacts = async (startDate: Date, endDate: Date) => {
+  const { s3Client, accountSid, bucketName } = await getContext();
+
+  const searchParams = getSearchParams(startDate, endDate);
+
+  const searchContactsResult = await contactApi.searchContacts(
```

Comment:
Could we send the contacts in the `Contact` shape rather than the `SearchContact` shape? We are hoping to deprecate the SearchContact shape when we do the search transcript work, so we want to keep it from appearing in any more public APIs in the meantime. 

It should just be a matter of adding a flag to the `searchContacts` method to specify whether we want to do the Contact -> SearchContact conversion or return the contacts as they come from the DB